boostnote.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=9.0.0
```

Comment:
It suppose to be `0.9.0`. Is it right?

And updating version manually is a little fragile. How about to create a script, which generate `boostnote.desktop` and `snap/snapcraft.yaml` with the version in `package.json`? And then, we can put the script into `postversion` of npm scripts so we can simply bump the app version with a single command, `npm version`.

Reply:
Oh right, thats a typo. As said, a helper script would be valuable yes.